pydicom/tests/test_filewriter.py

```diff
@@ -990,14 +990,14 @@ def test_ambiguous_element_in_sequence_explicit_using_index(self):
         fp = BytesIO()
         ds.save_as(fp, write_like_original=True)
         ds = dcmread(fp, force=True)
-        assert 'US' == ds[0x00283000][0][0x00283002].VR
+        assert 'US' == ds[0x00283000].value[0][0x00283002].VR
```

Comment:
Now that I've done this I can't help but feel this is uglier

Reply:
Should I revert and we keep as is?

Reply:
Agreed. I wasn't that happy while reviewing these changes.

Reply:
I've reverted the deprecation